ruby/.rubocop.yml

```diff
@@ -96,15 +96,6 @@ Naming/RescuedExceptionsVariableName:
 Naming/VariableNumber:
   Enabled: false
 
-Performance/RegexpMatch:
```

Comment:
These Cops were removed and added to the `rubocop-performance` gem. They were disabled, so remove them to get rid of the warning output